### PR TITLE
resource/alicloud_ess_eci_scaling_configuration: add instance_types.

### PR DESCRIPTION
**Pull request by fuliu-zln**
…nd upgrade api version

### CODE REVIEW

File: alicloud/resource_alicloud_ess_eci_scaling_configuration.go [hunk 1]
```diff
@@ -411,6 +411,14 @@ func resourceAlicloudEssEciScalingConfiguration() *schema.Resource {
 					},
 				},
 			},
+			"instance_types": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem: &schema.Schema{
+					Type:     schema.TypeString,
+					MaxItems: 5,
```
Comment (ChenHanZhang) [inline alicloud/resource_alicloud_ess_eci_scaling_configuration.go:419]: 测一下是否会乱序，用例里三个item，测试两个部分交互item顺序

Comment (fuliu-zln) [reply alicloud/resource_alicloud_ess_eci_scaling_configuration.go:419]: done

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Pull request merged**